### PR TITLE
Do not emit .eh_frame section if using -Cpanic=abort

### PR DESCRIPTION
**Pull request by antoyo**

### CODE REVIEW

**Comment by bjorn3** [comment]
`.eh_frame` is necessary once exceptions reaching into panic=abort rust code starts to abort the program rather than being UB.

**Comment by antoyo** [comment]
@bjorn3: This is necessary for Rust for Linux.
How do you suggest we handle this?

**Comment by bjorn3** [comment]
It may be that this change is fine and once unwinding into `panic=abort` rust code will abort the presence of the landingpad will cause `.eh_frame` to be emitted by both cg_llvm and cg_gcc. When that happens whatever solution is chosen for cg_llvm may work for cg_gcc too.

File: build_sysroot/build_sysroot.sh [hunk 1]
```diff
@@ -27,4 +27,10 @@ fi
 
 # Copy files to sysroot
 mkdir -p sysroot/lib/rustlib/$TARGET_TRIPLE/lib/
+# TODO: copy src if asked by the user.
```
Comment (antoyo) [inline build_sysroot/build_sysroot.sh:30]: Don't forget this.

**Comment by antoyo** [comment]
@bjorn3: Do you think this should instead be set/unset depending on the value of `-Cforce-unwind-tables`?

**Pull request merged**